tests/test_hf_api.py

```diff
@@ -1832,21 +1833,17 @@ def test_model_info(self):
         self.assertIsInstance(model, ModelInfo)
         self.assertEqual(model.sha, DUMMY_MODEL_ID_REVISION_ONE_SPECIFIC_COMMIT)
 
-    # TODO; un-skip this test once it's fixed.
-    @unittest.skip(
-        "Security status is currently unreliable on the server endpoint, so this"
-        " test occasionally fails. Issue is tracked in"
-        " https://github.com/huggingface/huggingface_hub/issues/1002 and"
-        " https://github.com/huggingface/moon-landing/issues/3695. TODO: un-skip"
-        " this test once it's fixed."
-    )
     def test_model_info_with_security(self):
+        warnings.warn(
+            "this test might fail, `security_repo_status` object structure is currently not stable on server side."
+        )
```

Comment:
```suggestion
        # Note: this test might break in the future if `security_repo_status` object structure gets updated server-side (not yet fully stable)
```

I would add a comment in the test itself (for future ourselves) but not a proper `warnings` as someone running the test won't really read it or have something to do with it when running the tests locally.

Reply:
addressed in [13be909](https://github.com/huggingface/huggingface_hub/pull/2639/commits/13be909f9fc8308f4945455b8bedb00b86a8502a)!